web/src/composebox_typeahead.ts

```diff
@@ -1008,8 +1034,25 @@ export function content_highlighter_html(item: TypeaheadSuggestion): string | un
             return typeahead_helper.render_typeahead_item({primary: item.language});
         case "topic_jump":
             return typeahead_helper.render_typeahead_item({primary: item.message});
-        case "topic_list":
-            return typeahead_helper.render_typeahead_item({primary: item.topic});
+        case "topic_list": {
+            if (item.is_stream && item.streamData) {
+                item.streamData.rendered_description = "";
```

Comment:
Overwrite this earlier when you set up the `streamData` value. Building on my earlier suggestion:  `streamData: {...stream, type: "stream", rendered_description: ""}`